.circleci/config.yml

```diff
@@ -96,6 +96,13 @@ jobs:
             npm install --global npm@7
             npm ci
             npm run build
+      - run:
```

Comment:
I think this should be done in a separate job - treating the jsonschemas directory as a separate module

Reply:
I've fixed this